client/network.go

```diff
@@ -9,10 +9,11 @@ import (
 	"strings"
 
 	flag "github.com/docker/docker/pkg/mflag"
+	"github.com/docker/libnetwork/netlabel"
 )
 
 const (
-	nullNetType = "null"
+	defaultDriverType = "bridge"
```

Comment:
I prefer to have this patch as part of the `multi-net` PR. That is the most appropriate PR that can make bridge as the default driver.